service/templates/cloudformation/guest/vpc.yaml

```diff
@@ -0,0 +1,20 @@
+{{define "vpc"}}
+  VPC:
+    Type: AWS::EC2::VPC
+    Properties:
+      CidrBlock: {{ .CidrBlock }}
+      EnableDnsSupport: 'true'
+      EnableDnsHostnames: 'true'
+      Tags:
+      - Key: Name
+        Value: {{ .ClusterID }}
+      - Key: Installation
+        Value: {{ .InstallationName }}
```

Comment:
Once we're migrated I'd like us to look at tagging. I think we can add some tags to the stack when we create it with the SDK. They will then propagate to all resources. I'd like to use it for the k8s cloud provider tags but maybe for the installation too.

Reply:
Great idea, will make things very easy. btw we should sync with @pipo02mix for this, he proposed https://github.com/giantswarm/aws-operator/pull/599, maybe that PR could be updated to accomplish what you suggest once the migration is done.

Reply:
Definitely, @pipo02mix I've seen your PR and it's great you're sorting out tagging. It will be easier to do this after the cloud formation migration.